### PR TITLE
Package zarith.1.9

### PR DESCRIPTION
**Pull request by antoinemine**
### `zarith.1.9`
Implements arithmetic and logical operations over arbitrary-precision integers
The Zarith library implements arithmetic and logical operations over
arbitrary-precision integers. It uses GMP to efficiently implement
arithmetic over big integers. Small integers are represented as Caml
unboxed integers, for speed and space economy.

---
* Homepage: https://github.com/ocaml/Zarith
* Source repo: git+https://github.com/ocaml/Zarith.git
* Bug tracker: https://github.com/ocaml/Zarith/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by Leonidas-from-XIV** [comment]
It looks like the `configure` script, line 258 is invalid and fails.

**Comment by antoinemine** [comment]
I see... It looks like we added some bash-specific code in ./configure, but Debian/Ubuntu interprets #!/bin/sh in a strict way (it works on my Gentoo, so I did not catch it). I could replace it with a sed code or force #!/bin/bash, not sure which one is the most portable. The bash-specific code is used only for Cygwin btw.

**Comment by Leonidas-from-XIV** [comment]
IMHO it would make the most sense to make a patch release with of zarith 1.9.1 or so to set that to `#!/bin/bash`. I don't think it is a great idea to do some sed trickery in OPAM since that only benefits OPAM users, but people installing the tarball on Debian or Ubuntu (e.g. packagers) will run into the exact same issue.

Debian and Ubuntu use `dash` as `/bin/sh` so they fail on bashisms, but the CI runs on Debian and also triggers that line so it doesn't seem to exclusively affect Cygwin.

**Comment by hhugo** [comment]
will fix https://github.com/ocaml/Zarith/issues/54

File: packages/zarith/zarith.1.9/opam [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  ] {os = "macos"}
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove:  ["ocamlfind" "remove" "zarith"]
+depends: [
+  "ocaml"
+  "ocamlfind"
```
Comment (kit-ty-kate) [inline packages/zarith/zarith.1.9/opam]:
```suggestion
  "ocamlfind" {build}
```

**Comment by kit-ty-kate** [comment]
The `remove` field is unnecessary since opam 2.0. Could you return the fix in your opam file upstream?

**Comment by antoinemine** [comment]
Thanks for the feedback. I've made a 1.9.1 bug-fix release and opam-publish-ed it as #14753

**Comment by kit-ty-kate** [comment]
This one can be closed then.

**Pull request closed without merging**